code/game/machinery/computer/guestpass.dm

```diff
@@ -91,7 +91,7 @@
 
 	if(giver)
 		data["giver"] = !!giver
-		data["giver_name"] = giver.rank || giver.assignment || SSjobs.get_by_path(giver.job_access_type).title
+		data["giver_name"] = giver.rank || giver.assignment || SSjobs.get_by_title(giver.rank)
```

Comment:
This line is redundant, `giver.rank` will short-circuit before it ever hits the SSjobs call. You can just delete `SSjobs.get_by_title(giver.rank)` entirely.